src/Mod/Fem/femtaskpanels/task_constraint_electrostaticpotential.py

```diff
@@ -111,7 +112,7 @@ def _initParamWidget(self):
             not self._paramWidget.capacitanceBodyBox.isChecked())
 
     def _applyWidgetChanges(self):
-        unit = "V"
+        # unit = "V"
```

Comment:
Remove completely?

Reply:
Yes, it no longer necessary, better remove it completely.